fuzzing/testdata/contracts/assertions/assert_allocate_too_much_memory.sol

```diff
@@ -0,0 +1,9 @@
+//This contract attempts to allocate an excessive amount of memory by creating an array with a length of 2^64 causing a panic.
+//PanicCodeAllocateTooMuchMemory = 0x41
```

Comment:
Can we make sure to add spaces after the "//" and before the comment. Sorry for being nitpicky but just want to make sure we follow the established pattern

Reply:
Same applies to all the test solidity files